packages/commonwealth/test/e2e/e2eRegular/customDomain.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, test } from '@playwright/test';
+import { createTestEntities, testChains } from '../hooks/e2eDbEntityHooks.spec';
+import { addAlchemyKey } from '../utils/e2eUtils';
+import { discussionTests } from './discussionsTest';
+
+// ssl proxy takes a while, so increase test timeout
+test.setTimeout(200000);
+
+test.use({ ignoreHTTPSErrors: true });
+
+test.beforeAll(async () => {
+  await createTestEntities();
+});
+
+test.beforeEach(async ({ page }) => {
+  // proxy can be slow to start up, so we wait for it.
+  await expect(async () => {
+    await page.goto(`https://${testChains[0]['custom_domain']}/`);
```

Comment:
Can we confirm anything more than that the page loads? For example, that it doesn't load the landing page but actually loads the community's page?

Reply:
@jnaviask The structure of this test is a little bit different than the rest, because the customDomain page is the same as the [community]/discussions page. So instead of copying everything over from our discussions test, I had pulled out the tests into a 3rd file to import into both this file and the discussions.spec.ts.

See the test.describe at the bottom which runs all the discussionsTests.ts with the new url (the custom domain url).